runner/ansible/roles/2.2.4/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.2.4'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # Check the pacemaker version IS
+    PACKAGE=$(rpm -qv corosync)
```

Comment:
I think we should use `sed` as in the pacemaker example here, to not have the whole `corosync` string.
And even use the same nomenclature like `VERSION_NUMBER` etc